python/graphstorm/dataloading/dataloading.py

```diff
@@ -131,6 +131,13 @@ def __next__(self):
         list of DGLGraph : the subgraph blocks for message passing.
         """
 
+    def __len__(self):
+        """ Return the length of the data loader
```

Comment:
what does length mean? the number of nodes/edges or the number of mini-batches?

Reply:
The number of batches. I have added more info into docstring.